QSB/TriggerSync/WorldObjects/QSBInflationTrigger.cs

```diff
@@ -67,6 +67,7 @@ private void StartCollapse()
 	{
 		var repelVolume = (WhiteHoleFluidVolume)TriggerOwner._repelVolume;
 		repelVolume.SetVolumeActivation(false);
+		QSBPlayerManager.ShowAllPlayers(); // seeing all players in big bang is funny so i want them visible
```

Comment:
feel free to leave comment about this if you dont want it

Reply:
disabled it because it's very distracting and usually just means you're seeing the very ending of the game with camera clipping and it looking jank. all the players are in the exact same spot - we make other players invisible for the statue cutscene for similar reasons

Reply:
fair enough. oh well